includes/discovery/sensors/state/apc.inc.php

```diff
@@ -230,11 +227,74 @@
         }
         $current = $data['memSensorsAlarmStatus'];
     }
-    $descr      = $data['memSensorsStatusSensorName'] . ' - ' . $data['memSensorsStatusSensorLocation'];
-    $divisor    = 1;
-    $multiplier = 1;
     if (is_numeric($current)) {
         discover_sensor($valid['sensor'], 'state', $device, $cur_oid, $state_name . '.' . $index, $state_name, $state_name, '1', '1', null, null, null, null, $current);
         create_sensor_to_state_index($device, $state_name, $state_name . '.' . $index);
     }
 }
+
+foreach ($pre_cache['r2pdu_humidity_temp_table'] as $index => $data) {
+    if ($data['rPDU2SensorTempHumidityStatusType'] != 'notInstalled') {
+        if ($data['rPDU2SensorTempHumidityStatusTempStatus']) {
+            $cur_oid        = '.1.3.6.1.4.1.318.1.1.26.10.2.2.1.9.' . $index;
+            $state_name     = $data['rPDU2SensorTempHumidityStatusName'] . ' Temperature Probe Status';
+            $state_index_id = create_state_index($state_name);
```

Comment:
This is the old style for creating state indexes.  Check here: https://docs.librenms.org/#Developing/Sensor-State-Support/